src/config/dynamic-theme-fixes.config

```diff
@@ -10109,6 +10109,16 @@ INVERT
 
 ================================
 
+*.mitx.*
+
+CSS
+img {
+	border-color: currentcolor;
```

Comment:
Is this needed?

Reply:
I think this is the selector that works to find the images that are problematic

Reply:
But why changing the border color?

Reply:
Ok seems it works without that. Sorry, I don't know anything about CSS, I just tried a couple of things until one worked.